legal-api/src/legal_api/services/filings/validations/continuation_in.py

```diff
@@ -159,10 +159,12 @@ def validate_business_in_colin(filing_json: dict, filing_type: str) -> list:
     msg = []
     business_identifier_path = f'/filing/{filing_type}/business/identifier'
     business_legal_name_path = f'/filing/{filing_type}/business/legalName'
+    business_founding_date_path = f'/filing/{filing_type}/business/foundingDate'
 
     if filing_json['filing'][filing_type].get('business'):
         identifier = filing_json['filing'][filing_type]['business']['identifier']
         legal_name = filing_json['filing'][filing_type]['business']['legalName']
+        founding_date = filing_json['filing'][filing_type]['business']['foundingDate']
```

Comment:
```suggestion
        legal_name = filing_json['filing'][filing_type]['business'].get('legalName')
        founding_date = filing_json['filing'][filing_type]['business'].get('foundingDate')
```
by this way it wont error out

Reply:
Thanks, will update